test/e2e/service_test.go

```diff
@@ -40,6 +40,7 @@ func TestService(t *testing.T) {
 
 	t.Log("create hello service duplicate and get service already exists error")
 	test.serviceCreate(t, r, "hello")
+	test.serviceCreatePrivate(t, r, "hello")
```

Comment:
cause of
```
    service_test.go:41: create hello service duplicate and get service already exists error
    cli.go:70: ERROR: cannot create service 'hello' in namespace 'kne2etests6' because the service already exists and no --force option was given
```
lets delete  earlier service creation

Reply:
OK. Changed it a bit.